node-runner-cli/commands/dockercommand.py

```diff
@@ -120,13 +121,19 @@ def config(args):
 
 
 @dockercommand([
+    argument("-a", "--autoapprove", help="Pass this option to run without any prompts. "
+                                         "Use this for automation purpose only", action="store_true"),
+    argument("-aue", "--advanceduserenvs",
```

Comment:
Parameter advanced user doesn't seem right.  As this is specific to extra config, probably `extranodeconfig` or `additionalnodeconfig` might be better name